### PR TITLE
Add the correct dependencies for samples - v2

### PR DESCRIPTION
**Pull request by cabelitos**
Changes since v1:

* Added commits 977b88a and 1f3232b

### CODE REVIEW

File: data/scripts/check-samples-dependencies.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+def loadBlackList(path):
+    with open(path, 'r') as f:
+        return json.load(f)
+
+if __name__ == "__main__":
+    exitWithError = False
+
+    if len(sys.argv) < 3:
+        print('Usage: ./check-sample-dependencies samples_root_dir black_list_dir')
```
Comment (bdilly) [inline data/scripts/check-samples-dependencies.py]: mmm... here you say black_list_dir, but actually you're waiting for apath for a json file, right?

Comment (cabelitos) [inline data/scripts/check-samples-dependencies.py]: Indeed I will change that.

File: data/jsons/samples_blacklist.json [hunk 2]
```diff
@@ -0,0 +1 @@
+["http-composed.fbp"]
```
Comment (bdilly) [inline data/jsons/samples_blacklist.json]: maybe you could name it as samples_dependency_check_skip_list or anything more descriptive than just a blacklist file on our data dir.

Comment (cabelitos) [inline data/jsons/samples_blacklist.json]: Will do. Thank you

**Comment by cabelitos** [comment]
merged

**Pull request closed without merging**